docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+***************
+Getting Started
+***************
+
+This chapter reviews some important things to consider before starting
+a METplus project.
+
+Questions to Consider
+=====================
+
+Questions to ask when applying METplus to a new testing and evaluation project
+------------------------------------------------------------------------------
+
+If a user is new to developing a verification or evaluation system or
+to using METplus, there are questions that should be considered to help
+determine which tools to use and how to set up METplus.
+
+* First and foremost, what are the questions need to be answered
+  with this testing and evaluation project?
+  
+* What type of forecasts and type of observations will be used and how
```

Comment:
Reverse the can/should with they -- "...type of observations will be used and how **can/should they** be matched?"

Reply:
Done